plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/Root.tsx

```diff
@@ -65,44 +66,49 @@ const InnerRoot = ({
   const headerId = `${hookId}-header`;
 
   return (
-    <CoderAuthWrapper type="card">
```

Comment:
This was one of the big problems with the previous design: you had a Card wrapper, but then inside it, you had another card.

Not terrible, but the problem was that you had to make sure that the auth card wrapper and the main content card wrapper were styled exactly the same and had to stay in sync. Obviously not great, so I refactored it so that there's only one source of the card styling now